nectar/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   New command to migrate the database to a new format;
     Use `nectar migrate-db status` to check if migration is needed;
     If so, backup your data and then execute `nectar migrate-db run` to proceed with the migration.
+-  **Config file breaking change**: Introduced the possibility to use a static rate instead of Kraken mid market rate.
```

Comment:
If you fancy, you could simplify the e2e tests now because we no longer need to mock the kraken API and as such you can get rid of our beloved `FakeTreasuryService` and directly specify the fixed rate at the top of the test files using the override functionality :)